kafka/4.4/modules/ROOT/pages/kafka-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,145 @@
+= Apache Kafka 4.x - Upgrade and Migrate - Mule 4
+:page-aliases: connectors::kafka/kafka-connector-upgrade-migrate.adoc
+
+Upgrade Anypoint Connector for Apache Kafka (Apache Kafka Connector) to version 4.3.
```

Comment:
I am confused: the title says 4.x but the intro sentence says 4.3.
Is it Kafka 4.3 or Mule 4.3?

Reply:
Oh, is it "upgrade Kafka 4.x (any *Kafka* version 4.whatever) to *Mule* 4.3?